client/globalLobby/GlobalLobbyWidget.cpp

```diff
@@ -38,7 +38,13 @@ GlobalLobbyWidget::GlobalLobbyWidget(GlobalLobbyWindow * window)
 {
 	addCallback("closeWindow", [](int) { GH.windows().popWindows(1); });
 	addCallback("sendMessage", [this](int) { this->window->doSendChatMessage(); });
-	addCallback("createGameRoom", [this](int) { if (!CSH->inGame()) this->window->doCreateGameRoom(); });//TODO: button should be blocked instead
+	addCallback("createGameRoom", [this](int) {
+#ifndef VCMI_HTML5_BUILD
+		//TODO: button should be blocked instead
+		if (!CSH->inGame()) this->window->doCreateGameRoom();
+#endif
+	});
+
```

Comment:
No wonders that we've started getting reports that lobby is broken...

Reply:
:rofl:

Reply:
Actually, I wanted to hide this button, but then I saw this TODO and thought it was okay. :)